client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	Painting: i18n.translate( 'Painting' ),
+	Paranormal: i18n.translate( 'Paranormal' ),
+	Parenting: i18n.translate( 'Parenting' ),
+	Parties: i18n.translate( 'Parties' ),
+	'Party promotion': i18n.translate( 'Party promotion' ),
+	'Performance art': i18n.translate( 'Performance art' ),
+	Performer: i18n.translate( 'Performer' ),
+	Perfume: i18n.translate( 'Perfume' ),
+	Personal: i18n.translate( 'Personal' ),
+	'Personal blog': i18n.translate( 'Personal blog' ),
+	'Personal development': i18n.translate( 'Personal development' ),
+	'Personal experiences': i18n.translate( 'Personal experiences' ),
+	'Personal finance': i18n.translate( 'Personal finance' ),
+	'Personal issues': i18n.translate( 'Personal issues' ),
+	'Personal trainer': i18n.translate( 'Personal trainer' ),
+	'Personal training': i18n.translate( 'Personal training' ),
+	'Personal website': i18n.translate( 'Personal website' ),
+	'Pet services': i18n.translate( 'Pet services' ),
+	Pets: i18n.translate( 'Pets' ),
+	Philosopher: i18n.translate( 'Philosopher' ),
+	Philosophy: i18n.translate( 'Philosophy' ),
+	Photocopier: i18n.translate( 'Photocopier' ),
+	'Photocopy repairs': i18n.translate( 'Photocopy repairs' ),
+	Photographer: i18n.translate( 'Photographer' ),
+	Photography: i18n.translate( 'Photography' ),
+	Photos: i18n.translate( 'Photos' ),
+	'Physical therapy': i18n.translate( 'Physical therapy' ),
+	Physiotherapy: i18n.translate( 'Physiotherapy' ),
+	'Pizza shop': i18n.translate( 'Pizza shop' ),
+	Plumber: i18n.translate( 'Plumber' ),
+	Podcast: i18n.translate( 'Podcast' ),
+	Poems: i18n.translate( 'Poems' ),
+	Poetry: i18n.translate( 'Poetry' ),
+	Pokémon: i18n.translate( 'Pokémon' ),
+	'Polical activisim': i18n.translate( 'Polical activisim' ),
+	'Policial candidate': i18n.translate( 'Policial candidate' ),
+	'Political analysis': i18n.translate( 'Political analysis' ),
+	Politics: i18n.translate( 'Politics' ),
+	Pollution: i18n.translate( 'Pollution' ),
+	Portfolio: i18n.translate( 'Portfolio' ),
+	'Portrait artist': i18n.translate( 'Portrait artist' ),
+	Pregnancy: i18n.translate( 'Pregnancy' ),
+	Presentations: i18n.translate( 'Presentations' ),
+	'Print design': i18n.translate( 'Print design' ),
+	Printing: i18n.translate( 'Printing' ),
+	'Private consulting': i18n.translate( 'Private consulting' ),
+	Product: i18n.translate( 'Product' ),
+	'Product marketing': i18n.translate( 'Product marketing' ),
+	'Product reviews': i18n.translate( 'Product reviews' ),
+	'Product sales': i18n.translate( 'Product sales' ),
+	'Professional cleaning service': i18n.translate( 'Professional cleaning service' ),
+	'Professional portfolio': i18n.translate( 'Professional portfolio' ),
+	'Professional profile': i18n.translate( 'Professional profile' ),
+	'Professional services': i18n.translate( 'Professional services' ),
+	'Promotional marketing': i18n.translate( 'Promotional marketing' ),
+	'Promotional products': i18n.translate( 'Promotional products' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Promotions' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).